### DISCUSSION TITLE
Moving from one Domain to another

### DISCUSSION THREAD

**Discussion by adminradio** [atrocore/atrocore#624] [Q&A]
Hi,
what is needed and which files have to be edited to move from one domain to another? Everything is the same, except the domain name changes. 

We have moved from `a.domain.tld` to `b.domain.tld` but getting only 400 and 500 Errors. We edited the obvious configuration values like `SiteURL` - but no luck so far. Any tips?

Thank you.

**Comment by atrolex** [comment]
Please check:
1) are there problems with file and folder permissions?
2) host configuration – is rewriting via .htaccess activated and working?
3) is cronjob running?
